scripts/updateDocs.sh

```diff
@@ -0,0 +1,42 @@
+set -eu
+
+GIT_ROOT="$(git rev-parse --show-toplevel)"
+cd $GIT_ROOT
+
+# Determine the version
+BRANCH="$(git branch | grep \* | cut -d ' ' -f2)"
+if [ "${BRANCH}" == 'develop' ]
+  then
+    VERSION='develop'
+  else
+    VERSION=v`cat VERSION`
+fi
+
+# Checkout gh-pages and pull the docs over from the original branch
+git checkout gh-pages
```

Comment:
Insert a git pull here so stuff is up to date. Otherwise you will get conflicts.